added a section called Support, with the first entry Technology Preview (#234)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/style_guidelines/support.adoc

[[support]]
= Support

[discrete]
== Technology Preview

> Technology Preview documentation guidelines apply to any form of early access feature that is not fully supported. Documentation for a Technology Preview feature might be incomplete or include only basic install and configuration information.

Reviewer: ```suggestion
Technology Preview documentation guidelines apply to any form of early access feature that is not fully supported. Documentation for a Technology Preview feature might be incomplete or include only basic installation and configuration information.
```

* Add an admonition labeled IMPORTANT at the beginning of the Technology Preview content.
* Use initial uppercase capitalization, that is, Technology Preview.

> * Include the Technology Preview feature in the Release Notes.

Reviewer: ```suggestion
* Include a brief description of the Technology Preview feature in the Release Notes.
```

> * Maintain a list of features that are currently in Technology Preview status in the Release

Reviewer: ```suggestion
* Maintain a list of features that are currently in Technology Preview status in the Release Notes.
```
typo

When a minor release becomes generally available, remove the IMPORTANT admonition from the Release Notes and any other document that includes content about the feature.

> Use the following template. Replace <feature> with the feature name:

Reviewer: ```suggestion
Use the following template. Replace _<feature_name>_ with the feature name:
```
Per SSG: italicize user-replaced values. Also consider using the format `feature_name` to be consistent with the SSG formatting guidance. Thanks!

====
IMPORTANT
====

> <feature> is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.

Reviewer: ```suggestion
_<feature_name>_ is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
```

[[support]]
= Support

[discrete]

> == Technology Preview

Author: done

> Technology Preview documentation guidelines apply to any form of early access feature that is not fully supported. Documentation for a Technology Preview feature might be incomplete or include only basic installation and configuration information.

Reviewer 2: Is this first sentence a true statement? I know that we have other types of non-supported levels, like dev preview. So I don't think we necessarily want people to label things TP if they're actually some other level. 

I'd recommend removing this first sentence and starting with a definition of what TP is, then leading into the guidelines. Maybe something like:

```suggestion
Technology Preview features provide early access to upcoming product innovations, enabling customers to test functionality and provide feedback during the development process. However, these features are not fully supported. Documentation for a Technology Preview feature might be incomplete or include only basic installation and configuration information.

When documenting a Technology Preview feature, follow these guidelines:
```

* Add an admonition labeled IMPORTANT at the beginning of the Technology Preview content.

> * Use initial uppercase capitalization, that is, Technology Preview.

Reviewer 2: ```suggestion
* Use initial uppercase capitalization, that is, "Technology Preview".
```

> * Include a brief description of the Technology Preview feature in the Release Notes.

Author: done

* Maintain a list of features that are currently in Technology Preview status in the Release Notes.

> When a minor release becomes generally available, remove the IMPORTANT admonition from the Release Notes and any other document that includes content about the feature.

Reviewer 2: Shouldn't this say when the feature becomes GA? Like:

```suggestion
When the Technology Preview feature becomes generally available, remove the IMPORTANT admonition from the release notes and any other document that includes content about the feature.
```

Use the following template. Replace _<feature_name>_ with the feature name:

[IMPORTANT]
====
_<feature_name>_ is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
====

> See link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope] for more information about the support scope of Red Hat Technology Preview features.

Reviewer 2: Can you move this statement inside the IMPORTANT note? It's part of what people should actually be including in their docs.

Though actually when I look at it again, I believe the exact wording from the template is:

```suggestion
For more information about the support scope of Red Hat Technology Preview
features, see https://access.redhat.com/support/offerings/techpreview/.
```

[[support]]
= Support

[[technology-preview-guidance]]
== Technology Preview

Technology Preview features provide early access to upcoming product innovations, enabling customers to test functionality and provide feedback during the development process. However, these features are not fully supported. Documentation for a Technology Preview feature might be incomplete or include only basic installation and configuration information.

When documenting a Technology Preview feature, follow these guidelines:

* Add an admonition labeled IMPORTANT at the beginning of the Technology Preview content.
* Use initial uppercase capitalization, that is, Technology Preview.
* Include a brief description of the Technology Preview feature in the release notes.
* Maintain a list of features that are currently in Technology Preview status in the release notes.

> 

Reviewer 3: ```suggestion
* Never use "supported as a Technology Preview", avoid "support" in TP descriptions if possible. Instead, use neutral words like: available, provide, capability, functionality, implement, enable,...
```

When the Technology Preview feature becomes generally available, remove the IMPORTANT admonition from the release notes and any other document that includes content about the feature.

Use the following template. Replace _<feature_name>_ with the feature name:

[IMPORTANT]
====

> _<feature_name>_ is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.

Reviewer 3: ```suggestion
_<feature_name>_ is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
```

====

> For more information about the support scope of Red Hat Technology Preview

Author: @bergerhoffer and @bburt-rh, I made the suggested change to the ID

[[support]]
= Support

[[technology-preview-guidance]]
== Technology Preview

Technology Preview features provide early access to upcoming product innovations, enabling customers to test functionality and provide feedback during the development process. However, these features are not fully supported. Documentation for a Technology Preview feature might be incomplete or include only basic installation and configuration information.

When documenting a Technology Preview feature, follow these guidelines:

> * Add an admonition labeled IMPORTANT at the beginning of the Technology Preview content.

Reviewer 2: We might want to mention here about the template. Maybe something like: 

```suggestion
* Add an admonition labeled IMPORTANT at the beginning of the Technology Preview content and include the template text.
```

* Use initial uppercase capitalization, that is, Technology Preview.
* Include a brief description of the Technology Preview feature in the release notes.
* Maintain a list of features that are currently in Technology Preview status in the release notes.

> * Never use "supported as a Technology Preview", and avoid using "support" in Technology Preview descriptions. Instead, use neutral words like "available", "provide", "capability", "functionality", "implement", "enable", and so on.

Reviewer 2: I also agree with @bburt-rh that this list of 6 examples is too long. I'd recommend cutting it down to "available", "provide" and maybe one other one.

> * Never use "supported as a Technology Preview", and avoid using "support" in Technology Preview descriptions. Instead, use neutral words like "available", "provide", "capability", "functionality", "implement", "enable", and so on.

Reviewer 2: ```suggestion
* Never use the phrase "supported as a Technology Preview", and avoid using "support" in Technology Preview descriptions. Instead, use neutral words like "available", "provide", "capability", "functionality", "implement", "enable", and so on.
```

> * Never use "supported as a Technology Preview", avoid "support" in Technology Preeview descriptions if possible. Instead, use neutral words like "available", "provide", "capability", "functionality", "implement", "enable", and so on.

Reviewer: ```suggestion
* Never use "supported as a Technology Preview", and avoid using "support" in Technology Preview descriptions. Instead, use neutral words like "available", "provide", "capability", "functionality", "implement", "enable", and so on.
```
- Fix the comma splice.
- Fix the typo `Preeview`.
- Consider omitting `if possible`. To my ears, it's implied by saying `avoid`. But you make the call...
- The second sentence that starts with `Instead` feels like a non-sequitur. I don't understand how one would use these words as alternatives to using the word `support`. 
- Imo the list is too long as-is -- maybe consider only 2 or 3 examples and make sure that they can serve as alternatives to using the word`support`.

* When the Technology Preview feature becomes generally available, remove the IMPORTANT admonition from the release notes and any other document that includes content about the feature.

For more information about the support scope of Red Hat Technology Preview

> features, see link:https://access.redhat.com/support/offerings/techpreview/(Technology Preview Features Support Scope).

Reviewer: ```suggestion
features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
```
- Fix the incorrectly formatted external link

Use the following template. Replace _<feature_name>_ with the feature name:

[IMPORTANT]
====

> _<feature_name>_ is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].

Reviewer 2: @leswilliams44 Sorry to be picky, but to match the template, the last sentence needs to be its own paragraph. Can you please update?

Also, to match the template exactly, it shows the actual URL (https://access.redhat.com/support/offerings/techpreview/), and not the name of the page. This is probably something we should consider changing, but for now I think we should match what the template shows on the source.

```suggestion
_<feature_name>_ is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. 

For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[https://access.redhat.com/support/offerings/techpreview/].
```

> _<feature_name>_ is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see link: https://access.redhat.com/support/offerings/techpreview/(Technology Preview Features Support Scope).

Reviewer: ```suggestion
_<feature_name>_ is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
```
- Fix the incorrectly formatted external link

When documenting a Technology Preview feature, follow these guidelines:

* Add an admonition labeled IMPORTANT at the beginning of the Technology Preview content and include the template text.
* Use initial uppercase capitalization, that is, Technology Preview.
* Include a brief description of the Technology Preview feature in the release notes.
* Maintain a list of features that are currently in Technology Preview status in the release notes.
* Never use the phrase "supported as a Technology Preview", and avoid using "support" in Technology Preview descriptions. Instead, use neutral words like "available", "provide", "capability", and so on.
* When the Technology Preview feature becomes generally available, remove the IMPORTANT admonition from the release notes and any other document that includes content about the feature.

For more information about the support scope of Red Hat Technology Preview
features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].

> Use the following template. Replace _<feature_name>_ with the feature name:

Reviewer: @ndeevy @bergerhoffer @leswilliams44 - Sorry to be "that person" that keeps bringing up issues, but in looking over this over again, I was just wondering why we are including a second copy of the TP template here when we already have one in the "Release notes" section. Why not just xref the actual code template we want people to use in the RN section? 

And if there's a case to include the template here, shouldn't it be identical to the one in the "Release notes" section and formatted as code? The currently published template refers to `tech_preview_name` rather than `feature_name` as the user-defined variable.

Reviewer 4: @bburt-rh  It's good to be "that person" if it's valid :-)  and I think it is here. I agree it'd be better not to duplicate it.
To be honest I think the currently published entries for tech preview and deprecated+removed functionality in the release notes section don't really belong there. That subsection was originally for writing doc texts, which is not where the contents of that template would appear anyway right. Maybe that all (tech preview and deprecated) belongs in this new Support section?
I just looked at a few BZs from release notes that have doc texts, and they are just the usual, tech writer-edited descriptions of the feature plus '<x> is available as a Technology Preview'.

Reviewer: @ndeevy- Thanks!  @bergerhoffer is submitting a separate PR to address some of these concerns, so we can continue our discussion re that new PR.

[[support]]
= Support

> [[technology-preview]]

Reviewer: This ID needs to be changed to a unique value. Since this ID is already in use elsewhere in `t.adoc`, when I build, I get the following asciidoctor error:

`asciidoctor: WARNING: glossary_terms_conventions/general_conventions/t.adoc: line 50: id assigned to block already in use: technology-preview`

Reviewer 2: Good catch @bburt-rh! Could update to something like [[technology-preview-guidance]] @leswilliams44

Author: @bburt-rh and @bergerhoffer, I revised the ID.

Author: Is there one last item? I am not sure what it is.

Reviewer: Thanks @leswilliams44. This now builds correctly and lgtm.
